lib/generators/active_admin/resource/resource_generator.rb

```diff
@@ -1,11 +1,17 @@
+require 'active_admin/generators/boilerplate'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.